iroh-sync/src/net/codec.rs

```diff
@@ -406,26 +415,35 @@ mod tests {
                     .unwrap()
                     .collect::<Result<Vec<_>>>()
                     .unwrap();
-                println!(
-                    "alice has: {:#?}",
-                    alice_messages
-                        .iter()
-                        .map(|e| (e.author(), std::str::from_utf8(&e.key()).unwrap()))
-                        .collect::<Vec<_>>()
-                );
+                // println!(
+                //     "alice has {} messages: {:#?}",
+                //     alice_messages.len(),
+                //     alice_messages
+                //         .iter()
+                //         .map(|e| (e.author(), std::str::from_utf8(&e.key()).unwrap()))
+                //         .collect::<Vec<_>>()
+                // );
+                let bob_messages = bob_replica_store
+                    .get(bob_replica.namespace(), GetFilter::all())
+                    .unwrap()
+                    .collect::<Result<Vec<_>>>()
+                    .unwrap();
+                // println!(
+                //     "bob has {} messages: {:#?}",
+                //     bob_messages.len(),
+                //     bob_messages
+                //         .iter()
+                //         .map(|e| (e.author(), std::str::from_utf8(&e.key()).unwrap()))
+                //         .collect::<Vec<_>>()
+                // );
+
                 assert_eq!(
                     alice_messages.len(),
                     2 * num_messages * num_authors,
                     "alice is missing messages"
                 );
-
                 assert_eq!(
-                    bob_replica_store
-                        .get(bob_replica.namespace(), GetFilter::all())
-                        .unwrap()
-                        .collect::<Result<Vec<_>>>()
-                        .unwrap()
-                        .len(),
+                    bob_messages.len(),
```

Comment:
We should expand the test to make sure that not just the numbers, but also the actual messages are correct.